instrumentation/github.com/gocql/gocql/config.go

```diff
@@ -123,7 +123,7 @@ func WithConnectInstrumentation(enabled bool) TracedSessionOption {
 
 // ------------------------------------------ Private Functions
 
-func configure(options ...TracedSessionOption) *TracedSessionConfig {
+func newConfig(options ...TracedSessionOption) *TracedSessionConfig {
```

Comment:
Not related to these changes, but can we update this to match the config name?

```suggestion
func newTracedSessionConfig(options ...TracedSessionOption) *TracedSessionConfig {
```